invenio_query_parser/contrib/spires/parser.py

```diff
@@ -139,6 +139,18 @@ class SpiresSimpleQuery(UnaryRule):
     grammar = attr('op', [SpiresKeywordQuery, SpiresValueQuery])
 
 
+class RangeValue(LeafRule):
```

Comment:
Indeed we could. I just have the feeling that the two parsers should be separated, since they parse different grammars. And it's also more readable for someone who is not familiar with the parser (having all the nodes on one file). 

If we prefer to use the nodes from the `invenio syntax parser` it's very easy to convert it.